### ISSUE TITLE
Make it easier to access raw data for "Missing from JS" question

### ISSUE THREAD

**Issue by dfabulich** [StateOfJS/StateOfJS-2020#53]
It took quite a bit of effort to get to the bottom of #51, because there was no easy way to get access to the raw responses.

Specifically, the raw answers were not available here https://www.kaggle.com/sachag/state-of-js

It would be even better if the "Export" button on the opinions page would provide direct access to the raw responses, rather than just the buckets/counts which is what's there now.

**Comment by SachaG** [comment]
Yes, I'm not sure why the responses got left out from the export. 

As for exporting them straight from the survey, that would be cool but I don't think it's currently exposed by our API so it would require a bit of work…
